packages/x-date-pickers-pro/src/internal/hooks/useMultiInputRangeField/useMultiInputDateTimeRangeField.ts

```diff
@@ -91,7 +96,7 @@ export const useMultiInputDateTimeRangeField = <TDate, TChildProps extends {}>({
         validationError: validateDateTimeRange({
           adapter,
           value: newDateRange,
-          props: { ...sharedProps, value: newDateRange },
+          props: sharedProps,
```

Comment:
I changed that because the `validateXXXX` methods never use `value` from `props` but directly from the `value` params, to TS was complaining that `value` does not exist on `props`.